controller/src/test/java/io/pravega/controller/server/eventProcessor/ConcurrentEventProcessorTest.java

```diff
@@ -75,38 +92,40 @@ public PositionInternal asImpl() {
         }
     }
 
-    ConcurrentEventProcessor<TestEvent, TestRequestHandler> processor;
-    int checkpoint = -1;
-    Map<Integer, Boolean> map = new HashMap<>();
-    CompletableFuture<Void> latch = new CompletableFuture<>();
-    CompletableFuture<Void> result = new CompletableFuture<>();
+    private int checkpoint = -1;
+    private Map<Integer, Boolean> map = new HashMap<>();
```

Comment:
checkpoint and map are both being referenced by multiple threads without synchronization.